### ISSUE TITLE
Add an optional way to prevent docstring consumption 

### ISSUE THREAD

**Issue by jubbers** [kangalioo/poise#237]
Currently, slash command docstrings are consumed by Poise's slash command macro to be used for Discord helper text content. While this is expected behavior, it prevents packages from using that same info from being consumed by `cargo doc` to generate documentation for commands built with the slash command macro. 

It would be fantastic to have the option to have some delineation between the two types of comments to allow `cargo doc` to generate documentation as usual or in a slightly adjusted way. 

TL;DR: It would be nice to be able to use docstrings for both help information in Discord and for `cargo doc` outputs somehow.

**Comment by jubbers** [comment]
See also [the snippet here](https://github.com/serenity-rs/poise/blob/6167311216065b953c6dc64406e56a31e52cd9a1/macros/src/command/mod.rs#L338) for the where this is happening per Kangalio.
